app/services/onetime/revert_to_product_level_refund_policies.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Used to process select sellers manually
+# For the rest, we'll use `seller_refund_policy_disabled_for_all` feature flag to override User#refund_policy_enabled flag
+#
+
+class Onetime::RevertToProductLevelRefundPolicies < Onetime::Base
+  LAST_PROCESSED_ID_KEY = :last_processed_seller_for_revert_to_product_level_refund_policy_id
+
+  attr_reader :seller_ids, :invalid_seller_ids
+
+  def self.reset_last_processed_id
+    $redis.del(LAST_PROCESSED_ID_KEY)
+  end
+
+  def initialize(seller_ids: [])
```

Comment:
With this approach, the list of `seller_ids` can be generated at runtime, via Rails console.